conda/nd.recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 package:
-  name: dynd-python
+  name: dynd.nd
   version: 0.7.2
 
 build:
   number: {{environ.get('TRAVIS_BUILD_NUMBER', 0)}}    # [unix]
   number: {{environ.get('APPVEYOR_BUILD_NUMBER', 0)}}  # [win]
   rpaths:
```

Comment:
the rpaths here should actually no longer be necessary. If they still are for some reason, it's a bug and we should address it in some other way.